extensions/sql-migration/src/telemtery.ts

```diff
@@ -18,12 +18,17 @@ export enum TelemetryViews {
 	AssessmentsDialog = 'AssessmentsDialog',
 	MigrationCutoverDialog = 'MigrationCutoverDialog',
 	MigrationStatusDialog = 'MigrationStatusDialog',
-	AssessmentsPage = 'AssessmentsPage'
+	AssessmentsPage = 'AssessmentsPage',
+	FeedbackDialog = 'FeedbackDialog',
```

Comment:
Would suggest something like `SqlMigrationFeedbackDialog` so that it's easy to tell where it came from just by looking at the action name.